src/PKSim.Core/Services/PKAnalysesTask.cs

```diff
@@ -177,24 +178,78 @@ public GlobalPKAnalysis CalculateGlobalPKAnalysisFor(IEnumerable<Simulation> sim
          if (simulation == null)
             return globalPKAnalysis;
 
-         simulation.Compounds.Each(compound =>
+         // For a population simulation the pk parameters are calculated at run time so we retrieve the global
+         // pk parameters from the simulation
+         if (simulation is PopulationSimulation populationSimulation)
          {
-            var pkCalculationOptions = _pkCalculationOptionsFactory.CreateFor(simulation, compound.Name);
-            var peripheralVenousBloodColumn = simulation.PeripheralVenousBloodColumn(compound.Name);
-            var venousBloodColumn = simulation.VenousBloodColumn(compound.Name);
-            var compoundContainer = calculateGlobalPKAnalysisFor(simulation, compound, peripheralVenousBloodColumn, venousBloodColumn, pkCalculationOptions);
-            globalPKAnalysis.Add(compoundContainer);
-         });
+            // Global PK Parameters are stored in containers with the compound name as the first child of the root container.
+            // There will be one container per compound with PK parameters stored within
+            var compoundContainers = createCompoundPKParameterContainers(populationSimulation);
```

Comment:
For population simulations, calculating the GlobalPKAnalysis is just retrieving the quantity parameters from the populationSimulation.PKAnalyses.